internal/audit/conf.go

```diff
@@ -22,6 +22,10 @@ type Conf struct {
 type confHolder struct {
 	// Backend states which backend to use for Audits.
 	Backend string `yaml:"backend" conf:",example=local"`
+	// IncludeMetadataKeys defines which gRPC request metadata keys should be included in access logs.
+	IncludeMetadataKeys []string `yaml:"includeMetadataKeys" conf:",example=['content-type']"`
+	// ExcludeMetadataKeys defines which gRPC request metadata keys should be excluded from access logs. Takes precedence over includeMetadataKeys.
+	ExcludeMetadataKeys []string `yaml:"excludeMetadataKeys" conf:",example=['content-type']"`
```

Comment:
I missed this earlier. The example is not quite correct if we use the same key for both include and exclude.  

```suggestion
	ExcludeMetadataKeys []string `yaml:"excludeMetadataKeys" conf:",example=['authorization']"`
```